src/test/javascript/admin.controller.test.js

```diff
@@ -1,34 +1,47 @@
 describe("AdminController", function () {
```

Comment:
Same as above, the ticket task is to write Jasmine tests for the general.controller, this should be a separate ticket.